fixtures/framework_with_native_swift_macro/Tuist/Package.swift

```diff
@@ -5,5 +5,6 @@ let package = Package(
     name: "FrameworkWithNativeSwiftMacro",
     dependencies: [
         .package(url: "https://github.com/alschmut/StructBuilderMacro.git", .exact("0.2.0")),
+        .package(url: "https://github.com/pointfreeco/swift-composable-architecture", .exact("1.4.0"))
```

Comment:
I would bump it also in the `app_with_spm_dependencies` just in case

Reply:
Very good idea. I'll add it to the other fixture too.